src/agent/web.ts

```diff
@@ -416,4 +417,34 @@ export default class Web extends WebAPI {
   async listAgentsForTransfer(chat_id: string): Promise<AgentForTransfer[]> {
     return this.send("list_agents_for_transfer", { chat_id });
   }
+
+  /**
+   * Marks a customer as followed. As a result, the requester (an agent) will receive
+   * the info about all the changes related to that customer via pushes.
+   * @param id - ID of customer
+   */
+  async followCustomer(id: string): Promise<EmptyResponse> {
+    return this.send("follow_customer", { id });
+  }
+
+  /**
+   * Removes the agent from the list of customer's followers.
+   * @param id - ID of customer
+   */
+  async unfollowCustomer(id: string): Promise<EmptyResponse> {
+    return this.send("unfollow_customer", { id });
+  }
+
+  /**
+   * Returns the current routing status of each agent.
+   * @param groupIds - Array of group IDs.
+   */
+  async listRoutingStatuses(groupIds?: number[]): Promise<SetRoutingStatusResponse[]> {
+    const filters: { group_ids?: number[] } = {};
+    if (groupIds && groupIds?.length > 0) {
+      filters.group_ids = groupIds;
+    }
```

Comment:
i think we do not put such logic here in the sdk, check other similar places

Reply:
Ok, I found example:

```ts
  async listAgents(group_ids?: number[], fields?: string[]): Promise<Agent[]> {
    return this.send("list_agents", { fields, filters: { group_ids } });
  }
```

Changed to remain consistent with the rest of the codebase.